### DISCUSSION TITLE
VoodooI2c Bug or Ventura beta?

### DISCUSSION THREAD

**Discussion by nitingaury** [nitingaury/Thinkpad-T470-Opencore#1] [Q&A]
Wake from sleep is broken when we use Touchscreen in Ventura beta, The kext loads and touchscreen works fine but the system restarts after wake from sleep. Done USB Mapping but issue still persists... so disabled touch screen in mapping and wake works perfectly fine now..!? still the issue is unknown.. some one could help on this and update the same..?

**Comment by devboloji** [comment]
Hello brother, i want to talk to you about hackintosh . Can i have your id your any telegram or mail or social media.? So rey to talk to you here. But i don't have option.

**Comment by devboloji** [comment]
> 
> Wake from sleep is broken when we use Touchscreen in Ventura beta, The kext loads and touchscreen works fine but the system restarts after wake from sleep. Done USB Mapping but issue still persists... so disabled touch screen in mapping and wake works perfectly fine now..!? still the issue is unknown.. some one could help on this and update the same..?

May be you need to add **bootargs** to it to fix sleep.

**Reply by nitingaury** [reply]
iMessage me nitingaury@hotmail.com

**Comment by devboloji** [comment]
@nitingaury is it possible to disable Touch screen? Can we use only keyboard and trackpad only!?

**Reply by nitingaury** [reply]
You can disable touch screen by usb mapping, that will save you battery too

**Reply by devboloji** [reply]
Brother, i have 450s and i don't have touch screen but some people have. [One guy](https://github.com/devboloji/Lenovo-Thinkpad-T450s-Hackintosh-Opencore-Guide-Maintaining/issues/1#issue-1446867516) has issue with it. Can you me how to do it?

**Comment by ucupsz** [comment]
hi nitingaury, 
just want to say thank you for sharing the EFI. i use your EFI for my monterey and work a bit for the wifi and bluetooth. works perfectly.
